app/src/main/java/com/geeksville/mesh/NodeInfo.kt

```diff
@@ -213,8 +214,9 @@ data class EnvironmentMetrics(
 data class NodeInfo(
     @PrimaryKey(autoGenerate = false)
     val num: Int, // This is immutable, and used as a key
+    @NonNull()
     @Embedded(prefix = "user_")
-    var user: MeshUser? = null,
+    var user: MeshUser,
```

Comment:
In Android, when working with AIDL and Parcelable objects, it is a common standard to use nullable types for properties, especially if the underlying data can be optional or can change over time. Most properties in `NodeInfo` are nullable and its more consistent to keep `user` nullable as well.

This would also be a breaking change for our API.